apps/voting/app/src/components/VoteActions.js

```diff
@@ -0,0 +1,271 @@
+              >
+                <IconCheck />
+              </div>
+            </div>
+            <div>
+              <div
+                css={`
+                  ${textStyle('body1')}
+                  margin-bottom: ${0.5 * GU}px;
+                `}
+              >
+                Vote success
+              </div>
+              <div
+                css={`
+                  ${textStyle('body2')}
+                  color: ${theme.surfaceContentSecondary};
+                `}
+              >
+                You voted{' '}
+                <span
+                  css={`
+                    color: ${theme.surfaceContent};
+                    font-weight: 600;
+                    text-transform: uppercase;
+                  `}
+                >
+                  {connectedAccountVote === VOTE_YEA ? 'yes' : 'no'}
+                </span>{' '}
+                with{' '}
+                <span
+                  css={`
+                    color: ${theme.surfaceContent};
+                    font-weight: 600;
+                  `}
+                >
+                  {userBalance === -1 ? '…' : userBalance} {tokenSymbol}
+                </span>
+                .
+              </div>
+            </div>
+          </div>
+        )}
+      </React.Fragment>
+    )
+  }
+
+  if (canUserVote && hasVoted && !changeVote) {
+    return (
+      <div>
+        <Button
+          mode="strong"
+          onClick={handleChangeVote}
+          wide
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          Change my vote
+        </Button>
+        <TokenReference
+          userBalance={userBalance}
+          tokenSymbol={tokenSymbol}
+          snapshotBlock={snapshotBlock}
+          startDate={startDate}
+        />
+      </div>
+    )
+  }
+
+  if (canUserVote) {
+    return (
+      <div>
+        {connectedAccount ? (
+          <React.Fragment>
+            <Buttons onClickYes={onVoteYes} onClickNo={onVoteNo} />
+            <TokenReference
+              userBalance={userBalance}
+              tokenSymbol={tokenSymbol}
+              snapshotBlock={snapshotBlock}
+              startDate={startDate}
+            />
+          </React.Fragment>
+        ) : (
+          <div
+            css={`
+              border-radius: ${RADIUS}px;
+              background: ${theme.background};
+              padding: ${3.5 * GU}px ${10 * GU}px;
+              text-align: center;
+            `}
+          >
+            <div
+              css={`
+                ${textStyle('body1')};
+              `}
+            >
+              You must enable your account to vote on this proposal
+            </div>
+            <div
+              css={`
+                ${textStyle('body2')};
+                color: ${theme.surfaceContentSecondary};
+                margin-top: ${2 * GU}px;
+              `}
+            >
+              Connect to your Ethereum provider by clicking on the Enable
```

Comment:
Should we add the lightning icon here?

cc @dizzypaty Perhaps we could also add emphasis on the "Enable account"?

Reply:
Yes, the lighting icon should be there and I agree with making 'Enable account' strong